packages/components/src/button/style.scss

```diff
@@ -171,7 +189,6 @@
 	&:disabled,
 	&[aria-disabled="true"] {
 		cursor: default;
-		opacity: 0.3;
```

Comment:
This change causes all disabled buttons which aren't one of the WordPress-styled variants to appear the same as a non-disabled button.

For example, in a new post, the Undo/Redo buttons appear now at full opacity.

![image](https://user-images.githubusercontent.com/1779930/61630390-de33a600-ac87-11e9-990b-f13667b973e7.png)

Reply:
See #16769